content/blog/probot/index.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Making GitHub apps with Probot
+date: "2019-07-04"
+description: Making GitHub apps easier with Probot and Node.js
+---
+### GitHub Apps with Probot
+Ever see in any issues or Pull Request in GitHub where a bot comments on the PR/Issue or adds labels to the Pr/Issue? If you have they are generally termed as GitHub apps and can be made relatively easily.  
+Most of these apps are made by JavaScript(Node.js) using **Probot**, a framework for building apps for GitHub. Probot is written in Typescript, a superset of JavaScript.You can find more about in their [Website](https://probot.github.io). 
+
+ ---------
+
+ Here we'll make a **Simple Issue and Pull Request commentor** bot which is made and deployed on Glitch.
+
+ ###### Prerequisites
+- GitHub account
+- An account on Glitch using GitHub
+- Base Js knowledge like variables and strings and functions
+#### Making our App on Glitch
+Glitch is a free online code-editing platform. It sets up our environment will all the necessary packages and stuff. All you need to do is click this button👇 and you'll get a full working environment for your "Remixing".
+<!-- Remix Button -->
+<!-- Remix Button -->
+<a href="https://glitch.com/edit/#!/remix/twilight-marquis">
+  <img src="https://cdn.glitch.com/2bdfb3f8-05ef-4035-a06e-2043962a3a13%2Fremix%402x.png?1513093958726" alt="remix this" height="33">
+</a>
+
+Probot ships with an example of setting up an issue commentor bot in `index.js` for commenting on PRs we need to uncomment the `app.yml` file at line 38, written pull request removing the '#' will do the trick
+Next, we'll edit the `index.js` to the following 
+
+ ```js 
+//index.js
+ module.exports = app => {
+  // Your code here
+  app.log('Yay, the app was loaded!')
+  const issue=context.payload.issue //getting issue details
+  const user=issue.login.user //Getting the username
+  app.on('issues.opened', async context =>({
+    return context.github.issues.createComment(context.issue({ body: `Thanks @${user} for opening this issue!:tada:
+    You are awesome..` })
+)
+  })
+}
+```
+The above code is for a bot which comments on new issues. It mentions the user too. The *"Robotic"* characteristics start from the 4th line starting with `app.on` function the first argument passed is the action which the bot listens to. These actions are **webhook** actions of GitHub and you can learn more about from[Webhook Documentation](https://developer.github.com/webhooks/).    
+Probot uses node's `async` API for returning the data. We used *`* to enclose the string/comment body because we won't need to use escape characters and we can also interpolate [Template Literals](https://flaviocopes.com/javascript-template-literals/).         
```

Comment:
```suggestion
Probot uses Node's `async/await` API for returning the data. We used the *`* character to enclose the string/comment body because we won't need to use escape characters and we can also interpolate [Template Literals](https://flaviocopes.com/javascript-template-literals/).         
```